app/services/access_process_service.rb

```diff
@@ -0,0 +1,75 @@
+class AccessProcessService
+  include RBAC::Permissions
+
+  def initialize(opts = {})
+    @app_name = opts[:app_name] || 'approval'
```

Comment:
@hsong-rh Could this be ENV['APP_NAME']
https://github.com/ManageIQ/catalog-api/blob/ed1d457e9353aae515684d2445126e749d621088/lib/rbac/access.rb#L9